### PR TITLE
Feature/coac-theme

### PR DESCRIPTION
**Pull request by ghost**
in this PR:

* change the logo

* add fonts (Blogger Sans, Roboto, Gidole)

* add theme COAC and selecting for default

* add folder hydrogen in styles

* add styles for hydrogen appearance

### CODE REVIEW

File: assets/styles/hydrogen/_overrides.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+body {
+    font-family: $family-primary;
+}
+
+.ok-post {
+    box-shadow: $box-shawdow-ocac;
```
Comment (EdisonArango) [inline assets/styles/hydrogen/_overrides.scss]: You mispelled coac:

```suggestion
    box-shadow: $box-shadow-coac;
```

File: assets/styles/hydrogen/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+$family-primary: 'Gidole', sans-serif;
+$family-titles: Blogger Sans, sans-serif;
+$box-shawdow-ocac:  0 8px 8px 0 rgba(0,0,0,.05);
```
Comment (EdisonArango) [inline assets/styles/hydrogen/_variables.scss]:
```suggestion
$box-shadow-coac:  0 8px 8px 0 rgba(0,0,0,.05);
```

File: package-lock.json [hunk 3]
```diff
@@ -2343,6 +2343,15 @@
...
     },
+    "@types/cheerio": {
+      "version": "0.22.22",
+      "resolved": "https://registry.npmjs.org/@types/cheerio/-/cheerio-0.22.22.tgz",
+      "integrity": "sha512-05DYX4zU96IBfZFY+t3Mh88nlwSMtmmzSYaQkKN48T495VV1dkHSah6qYyDTN5ngaS0i0VonH37m+RuzSM0YiA==",
+      "dev": true,
+      "requires": {
+        "@types/node": "*"
+      }
+    },
```
Comment (EdisonArango) [inline package-lock.json]: Why was this file updated?

File: static/i18n/es.js [hunk 4]
```diff
@@ -1,3 +1,3 @@
 export default {
-  "name" : "Okuna"
+  "name" : "Coac"
```
Comment (EdisonArango) [inline static/i18n/es.js]:
```suggestion
  "name" : "COAC"
```

Comment (ghost) [reply static/i18n/es.js]: ¿could be a deleted this name?. it shows on the login page and I don't believe  that necessary

File: assets/styles/hydrogen/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+$family-primary: 'Gidole', sans-serif;
+$family-titles: Blogger Sans, sans-serif;
+$box-shawdow-coac:  0 8px 8px 0 rgba(0,0,0,.05);
```
Comment (EdisonArango) [inline assets/styles/hydrogen/_variables.scss]: It's `shadow` instead of `shawdow`
```suggestion
$box-shadow-coac:  0 8px 8px 0 rgba(0,0,0,.05);
```

**Comment by EdisonArango** [comment]
Pushed to final repo

**Pull request closed without merging**